BootloaderCommonPkg/Include/Library/ConfigDataLib.h

```diff
@@ -59,11 +59,42 @@ typedef struct {
   //
   UINT8   HeaderLength;
   UINT8   Attribute;
+  union {
+    //
+    // Internal configuration data offset in DWORD from the start of data blob.
+    // This value is only valid in runtime.
+    //
+    UINT16  InternalDataOffset;
+    //
+    // Security version number
+    // This is available only in flash. It would be overwritten by CDATA_BLOB.InternalDataOffset in runtime
+    //
+    UINT8   Svn;
+    // Reserved
+    UINT8   Rsvd;

```

Comment:
Thanks Guo. Yes, SVN and RSVD would point to same location.
Specifying bits would also point to same value.  We can use struct so that rsvd can be used for future.

struct{
   UINT8   Svn;
    UINT8   Rsvd;
}

Reply:
Done